boomerang.js

```diff
@@ -1428,6 +1432,7 @@ BOOMR_check_doc_domain();
 
 			impl.vars["ua.plt"] = navigator.platform;
 			impl.vars["ua.vnd"] = navigator.vendor;
+			impl.vars["ua.raw"] = navigator.userAgent;
```

Comment:
don't add this.  It makes the beacon very large and is unnecessary since you can already get the user agent from the `User-Agent` HTTP request header.